src/WordPress/Blueprints/Model/DataClass/UnzipStep.php

```diff
@@ -2,60 +2,66 @@
 
 namespace WordPress\Blueprints\Model\DataClass;
 
-class UnzipStep implements StepDefinitionInterface
-{
+class UnzipStep implements StepDefinitionInterface {
+
 	public const DISCRIMINATOR = 'unzip';
 
 	/** @var Progress */
-	public $progress;
+	public Progress $progress;
 
 	/** @var bool */
-	public $continueOnError;
+	public bool $continueOnError;
 
-	/** @var string */
-	public $step = 'unzip';
+	/**
+	 * Step
+	 *
+	 * @var string
+	 */
+	public string $step = 'unzip';
+
+	/**
+	 * The file to extract
+	 *
+	 * @var string|ResourceDefinitionInterface
+	 */
 
-	/** @var string|ResourceDefinitionInterface */
-	public $zipFile;
+	// @TODO: Union types are PHP >= 8.0. Fix needed.
```

Comment:
Why add that type, then?

Reply:
I come from a Java background. I guess, I still have to get used to fields being only partially typed...